Topics/User_Experience.md

```diff
@@ -21,9 +21,18 @@ ___Established Norms___
 ___Color Theory___
 - [Colors Influence Choice](https://usabilitygeek.com/colour-user-experience-psychology/#:~:text=Colour%20plays%20a%20crucial%20role,and%20identified%20with%20your%20industry.): Users are shown certain colors to enforce actions. For example, red is commonly linked to aggressive or bad emotions and as such can be used as a cancel to dissuade people from refunding items.
 
+___Responsive Design___
+- [Responsive Design](https://devrix.com/tutorial/important-responsive-design/): Demonstrates important and benefits of responsive design, flexiblity they provide, easiblity for the consumers upon making the said software. 
+
 ___ Nielsen's 10 Usability Heuristics___
 - [Usability Heuristics](./User_Experience/Usability_Heuristics.md)
 
+### Additional UX
+
+There are some things that have become a crucial part in making a software: Universal Design Principles. 
+___Accessiblity Features___
+- [Universal Design Principles](https://www.buffalo.edu/access/help-and-support/topic3/universaldesignprinciples.html): This article provides the 7 Universal Design Principle that makes a software accessible to all users, which plays a important role in diversifying the User Experience. 
```

Comment:
Grammatical error, change "This article provides the 7 Universal Design Principle" to "This article provides the 7 Universal Design Principles".